coralproject/talk
# Only shows the Loading component when there is no search value

## What does this PR do?

removes Loading when there is a search value in admin/Community/People

## How do I test this PR?

- go to admin/community/people
- search something
- check that there is no 'flashing load comments' thingie coming up
